src/io/opendss.jl

```diff
@@ -355,17 +355,27 @@ function _dss2pmd_shunt!(pmd_data::Dict, dss_data::Dict, import_all::Bool)
             nconductors = pmd_data["conductors"]
             name, nodes = _parse_busname(defaults["bus1"])
 
-            Zbase = (pmd_data["basekv"] / sqrt(3.0))^2 * nconductors / pmd_data["baseMVA"]  # Use single-phase base impedance for each phase
-            # should be in 1MW Sbase, because make_per_unit will rescale to real power base later on already
-            # also, we want the total Sbase, not per phase
-            Zbase = Zbase*pmd_data["baseMVA"]/3
-            # should be  positive; the capacitor power reference has generator convention, not load
-            Gcap = Zbase * sum(defaults["kvar"]) / (nconductors * 1e3 * (pmd_data["basekv"] / sqrt(3.0))^2)
+            vnom_ln = defaults["kv"]
+            # 'kv' is specified as phase-to-phase for phases=2/3 (unsure for 4 and more)
+            if defaults["phases"] > 1
+                vnom_ln = vnom_ln/sqrt(3)
+            end
+            # 'kvar' is specified for all phases at once; we want the per-phase one, in MVar
+            qnom = (defaults["kvar"]/1E3)/defaults["phases"]
+            b_cap = qnom/vnom_ln^2
+            #  get the base addmittance, with a LN voltage base
+            Sbase = 1 # not yet pmd_data["baseMVA"] because this is done in _PMs.make_per_unit
+            Ybase_ln = Sbase/(pmd_data["basekv"]/sqrt(3))^2
+            # now convent b_cap to per unit
+            b_cap_pu = b_cap/Ybase_ln
+            #TODO clean this up when revisiting p.u. conversion
+            # this is needed because the shunt is non-dimensionalized at PM level later on
+            b_cap_pu = b_cap_pu
```

Comment:
Sorry, I'm not clear on the need for this line from the comment?

Reply:
This is a typo, I moved what that line was supposed to do up. Fixing it now